src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/IntegrationTestRuntime.kt

```diff
@@ -14,7 +14,6 @@ object IntegrationTestRuntime {
     init {
         ConfigurationFactory.setConfigurationFactory(LogConfigurationFactory(taskWorkspace))
         aws = Aws.Builder(Regions.EU_WEST_1)
-            .availabilityZoneFilter(Predicate { it.zoneName !in listOf("eu-west-1a") })
```

Comment:
I don't remember why we didn't change it before. AFAIR we wanted to provision all the instances in the same AZ to ensure both experiment and baseline run on similar network topology. CC: @dagguh

Reply:
Are you referring here to this particular changed line or to the entire changeset?

Currently we're not running baseline and experiment on the same AZ (it's randomly chosen) and my commit doesn't change it -- we'll still have different zones, but they will be assigned by AWS infra and not specified by us. Ensuring that we run everything on the same AZ is a different story and would require going even further in the initial direction which is considered by AWS folks a bad idea.

I propose to go with my change and see if it helps in terms of stability. If not, then we'll have an argument when opening a support case.

Reply:
The reason is in this diff in this PR:
https://github.com/atlassian/aws-infrastructure/pull/90/files#diff-a54501e4223a91aed36123deeaa1077dL8-L9

> Randomizing helps spread resources over a wider capacity pool.

If it actually doesn't help with capacity and provisioning, then we should indeed remove it.

Consistent AZs are not enforced anyway. We only try to keep the same `Network` across the `InfrastructureFormula`.